DEVELOPMENT.md

```diff
@@ -65,6 +65,11 @@ make run-ci V=1
 
 ## Running Tests
 
+To install the libraries required for testing:
+```bash
+pip install -e ".[test]"
```

Comment:
Not sure if this is adding any value.

Should be handled with `uv sync --group dev`